validator/accounts/v2/accounts_list.go

```diff
@@ -97,13 +98,15 @@ func listDirectKeymanagerAccounts(
 		fmt.Printf("%s %#x\n", au.BrightMagenta("[public key]").Bold(), pubKeys[i])
 
 		// Retrieve the account creation timestamp.
-		createdAtBytes, err := wallet.ReadFileAtPath(ctx, accountNames[i], direct.TimestampFileName)
+		keystoreFileName, err := wallet.FileNameAtPath(ctx, accountNames[i], direct.KeystoreFileName)
 		if err != nil {
-			return errors.Wrapf(err, "could not read file for account: %s", direct.TimestampFileName)
+			return errors.Wrapf(err, "could not get keystore file name for account: %s", accountNames[i])
 		}
-		unixTimestampStr, err := strconv.ParseInt(string(createdAtBytes), 10, 64)
+		timestampStart := strings.LastIndex(keystoreFileName, "-") + 1
```

Comment:
Being able to retrieve the timestamp from the file name should be its own helper func